charts/atlas-operator/templates/crds/crd.yaml

```diff
@@ -14,7 +14,14 @@ spec:
     singular: atlasmigration
   scope: Namespaced
   versions:
-  - name: v1alpha1
+  - additionalPrinterColumns:
+    - jsonPath: .status.conditions[?(@.type=="Ready")].status
+      name: Ready
+      type: string
+    - jsonPath: .status.conditions[?(@.type=="Ready")].reason
```

Comment:
Could we add a column for 'message'? It will allow us to review quickly by running 'Kubectl get atlasschemas'